### PR TITLE
Update refcache action to actually run the fix

### PR DESCRIPTION
**Pull request by cartermp**
It only runs the check - the update to the refcache isn't done.

### CODE REVIEW

**Review by theletterf — APPROVED** [review]

File: .github/workflows/pr-actions.yml [hunk 1]
```diff
@@ -104,6 +104,7 @@ jobs:
       - run: npm install --omit=optional
       - run: |
           npm run check:links
+          npm run test-and-fix
```
Comment (pellared) [inline .github/workflows/pr-actions.yml:107]: Is the previous step needed then?

Comment (svrnm) [reply .github/workflows/pr-actions.yml:107]: `test-and-fix` does much more than fixing the refcache (e.g. it also runs `fix:dict` and `fix:filenames`)

If I remember correctly `check:links` *should* update the refcache files except there is any issue going on.

Comment (cartermp) [reply .github/workflows/pr-actions.yml:107]: It doesn't appear to be updating anything, but `test-and-fix` does.

**Review by svrnm — APPROVED** [review]

File: .github/workflows/pr-actions.yml (hunk 1 above)
Comment (svrnm) [reply .github/workflows/pr-actions.yml:107]: that's odd ... but let's try it out then, I approved the PR

**Comment by cartermp** [comment]
Huh. I tried on a another branch and indeed the original check worked? Maybe github actions are just so slow that I never noticed.

**Pull request closed without merging**